rest-api/src/main/scala/za/co/absa/enceladus/rest_api/controllers/SchemaController.scala

```diff
@@ -146,9 +146,9 @@ class SchemaController @Autowired()(
     try {
       for {
         // the parsing of sparkStruct can fail, therefore we try to save it first before saving the attachment
-        update <- schemaService.schemaUpload(username, menasAttachment.refName, menasAttachment.refVersion - 1, sparkStruct)
+        (update, validation) <- schemaService.schemaUpload(username, menasAttachment.refName, menasAttachment.refVersion - 1, sparkStruct)
```

Comment:
Yes, it can, but I did this deliberately.

Ff you see `(update, _)`, you don't know what it was right away, apart from the fact that it is not utilized. This way, you can see that it is the `validation` returned from the method that is and the implementation visibly disregards it for V2 API (because the `schemaUpload` method has changed in signature to allow the validation in, but V2 never used it, so it does not use it now)

Reply:
👍 Then I would just add an explanatory comment - like in all cases when something little unexpected is done.

Reply:
@benedeki, I agree. That is why the explanatory comment is two lines below.
```scala
// v2 disregarding the validation; conforming to V2 Option[Entity] signature
```

Reply:
Sorry 😊